src/components/button/PillButton.stories.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { PillButton } from './Button';
+
```

Comment:
Your import statement is incorrect. If you change this to 
`import { PillButton } from './PillButton';`
this will work. Also I'd make a separate folder for pillbutton since it's its own component. Once you use template it's going to show up as a separate component on storybook anyways :)